test/tests/functional/pbs_pbsnodes.py

```diff
@@ -89,3 +89,21 @@ def test_pbsnodes_aS(self):
             self.assertIn(
                 hdr, out2['out'][0],
                 "header %s not found in output" % hdr)
+
+    def test_pbsnodes_av(self):
+        """
+        This verifies the values of last_used_time in 'pbsnodes -av'
+        result before and after server shutdown, once a job submitted.
+        """
+        j = Job(TEST_USER)
+        j.set_sleep_time(1)
+        jid = self.server.submit(j)
+        self.server.accounting_match("E;%s;" % jid)
+        val_before = self.server.status(NODE)[0]['last_used_time']
+        self.logger.info("Restarting server")
+        self.server.restart()
+        self.assertTrue(self.server.isUp(), 'Failed to restart Server Daemon')
+        val_after = self.server.status(NODE)[0]['last_used_time']
+        self.logger.info("Before: " + val_before + ". After: " + val_after)
```

Comment:
Suggestion: Either add a period at the end or remove the period from the middle.